ember-app/app/components/hb-column-crumb.js

```diff
@@ -5,11 +5,11 @@ var HbColumnCrumbComponent = Ember.Component.extend({
   classNames: ['crumb'],
   classNameBindings: ["stateClass", "isSelected:active:inactive", "indexClass"],
   isSelected: function(){
-    if(this.get("issue.data.state") === "closed"){
-      return this.get("visibleColumns.lastObject.data.name") === this.get("column.data.name");
-    }
-    return this.get("issue.data.current_state.name") === this.get("column.data.name");
-  }.property("issue.data.current_state.name", "column.data.name", "issue.data.state"),
+    var current_issue = this.get("issue");
+    return this.get("column.sortedIssues").find((issue)=> {
+      return issue.get("id") === current_issue.get("id");
```

Comment:
I doubt it, if anything it would be incredibly minuscule. The Ember syntax for that line would be more accurately:

`var current_id = this.get("issue.id");` 

I personally find writing methods with comparisons in the callback easier to read if the left and the right are as similar is possible.

To answer your final question this block _could_ also be written like so:

```javascript
return this.get("column.sortedIssues").find((issue)=> {
 return issue.get("id") === this.get("issue.id");
}.bind(this));
```

Which has a nice terseness to it but the `bind` method tends to trip people (and some versions of phantomjs) up.